flink-metrics/flink-metrics-influxdb/src/main/java/org/apache/flink/metrics/influxdb/InfluxdbReporterOptions.java

```diff
@@ -57,6 +59,15 @@
 		.defaultValue("")
 		.withDescription("(optional) the InfluxDB retention policy for metrics");
 
+	public static final ConfigOption<String>  CONSISTENCY = ConfigOptions
```

Comment:
ConfigOptions now support enums, which should simplify this PR a bit. the config docs generator also automatically extracts possible values, so the description can be slimmed down as well.

Reply:
I used ‘mvn package -Dgenerate-config-docs -pl flink-docs -am -nsu -DskipTests’ too generator docs， But It don't work， Any wrong ？

Reply:
I think Chensary would suggest to use `enumType` instead of current `defaultValue` and the reason why you not generate docs is because`InfluxdbReporterOptions` not included in `org.apache.flink.docs.configuration.ConfigOptionsDocGenerator` currently.

Please refine current parameters description in docs if you decide to refer to `Influxdb_reporter_coniguration.html`

Reply:
@zentol I think I have meet a error, Flink-docs check option key between promethus and influxdb, add throw a error when Ci. I have read the code,but I don't know how to reslove it, As jira https://issues.apache.org/jira/browse/FLINK-14855